pom.xml

```diff
@@ -1,9 +1,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<groupId>org.lightcouch</groupId>
+	<groupId>org.lightcouch.legacy.v1</groupId>
 	<artifactId>lightcouch</artifactId>
-	<version>0.1.4</version>
+	<version>0.1.4.1</version>
```

Comment:
POM changes that updates the **groupId** and **version**.

Reply:
Did we actually need a new version given the group is changed, making this effectively a different artifact?

nit: should the new group id be com.vi?

Reply:
> Did we actually need a new version given the group is changed, making this effectively a different artifact?

We don't. But we can't create a branch named `lightcouch-0.1.4` because a tag with that name already exists. My intention was to keep the version number (`0.1.4.1`) in tandem with the branch name (`lightcouch-0.1.4.1`).

> nit: should the new group id be com.vi?

Yes, like that.